ui/src/main/webapp/src/app/components/project-list/project-list.component.scss

```diff
@@ -72,6 +47,14 @@ a {
     .project-info:hover { background-color: rgb(222, 243, 255); }
 }
 
+/* Override Patternfly's middle alignment. */
+.list-view-pf.projects-list .list-group-item.project-info,
+.list-view-pf.projects-list .list-view-pf-description { align-items: flex-start; }
+
+/* Action buttons padding */
+.list-view-pf.projects-list .list-view-pf-actions span { margin-left: 1em; }
+.list-view-pf.projects-list .list-view-pf-actions span a i { font-size: 16pt; }
```

Comment:
Seriously, sticking with the size which is quite hard to nagate the pointer to is wrong, and doing so just because it's like some other UI is wrong approach. You'll see yourself.